torch/fx/graph.py

```diff
@@ -291,16 +291,15 @@ def __exit__(self, type, value, tb):
 
 class _node_list:
     def __init__(self, graph: "Graph", direction: str = "_next"):
```

Comment:
This `__init__` should be typed as a literal for static type checking:
```suggestion
    def __init__(self, graph: "Graph", direction: Literal["_prev", "_next"] = "_next"):